### PR TITLE
AEA-311 AEA-264 AEA-243 Bugfix/registry compatibility

### PR DESCRIPTION
**Pull request by panasevychol**
## Fixes

- item unpacking path on add fixed (added `vendor` subpath)
- logic command call changed to `aea login USERNAME --password=[password]`
- tests fixed
- docs updated

#578 

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../master/CONTRIBUTING.rst) doc
- [ ] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have checked that the documentation about the `aea cli` tool works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: aea/cli/common.py [hunk 1]
```diff
@@ -379,7 +379,7 @@ def try_get_item_target_path(
 
     :return: the item target path
     """
-    target_path = os.path.join(path, author_name, item_type_plural, item_name)
+    target_path = os.path.join(path, "vendor", author_name, item_type_plural, item_name)
```
Comment (DavidMinarsch) [inline aea/cli/common.py]: The function is called `try_get_item_target_path`. I am worried that this is ambiguous. It should probably be renamed to `try_get_vendorized_item_target_path` to make clear that this is only for items in the vendor folder.

File: aea/cli/registry/fetch.py [hunk 2]
```diff
@@ -44,6 +44,7 @@ def fetch_agent(ctx: Context, public_id: PublicId) -> None:
     target_folder = os.path.join(ctx.cwd, name)
     os.makedirs(target_folder, exist_ok=True)
 
+    click.echo('Fetching dependencies...')
```
Comment (DavidMinarsch) [inline aea/cli/registry/fetch.py]: I notice, that in some places we use click built in print functionality and in other places the logger.info. Can we unify this as part of this PR and use `click.echo` for all `logger.info`? @marcofavorito thoughts? I think `click.echo` might be preferable as it means the `info` bit is missing from the output.

Comment (panasevychol) [reply aea/cli/registry/fetch.py]: I would prefer `click.echo` for messages that should reach user. Logging probably sould be used for dev related output.

File: aea/cli/common.py (hunk 1 above)
Comment (panasevychol) [reply aea/cli/common.py]: But we came to a conclusion that target (the thing that we add/fetch) is always a published thing so it should go to `vendor` folder. Maybe that assumption is enough and we don't need that long naming in this case?)

File: aea/cli/registry/fetch.py (hunk 2 above)
Comment (DavidMinarsch) [reply aea/cli/registry/fetch.py]: Ok, so please apply `click.echo` throughout where `logger.info` is used atm. `logger.debug` of course remains.

File: aea/cli/common.py (hunk 1 above)
Comment (DavidMinarsch) [reply aea/cli/common.py]: Well but another developer might easily confuse it and use it wrongly, I think in this case it would help to replace the names

**Pull request merged**